db/migrate/20170925080041_create_skills.rb

```diff
@@ -0,0 +1,14 @@
+# Commit fot rubocop
```

Comment:
remove comment

Reply:
In this case I'll not pass rubocop. It shows this remark: "Missing top-level class documentation comment."

Reply:
Should I remove them?

Reply:
It looks like:
Errors on modified lines:
/home/stepan/Documents/Project/Ruby_Combats/db/migrate/20170927151536_create_skills.rb:1:1: C: Style/Documentation: Missing top-level class documentation comment.
So rubocop will forbid me to create commit if this line will be deleted.